compliance/collection/nodescanv2/nodescan.go

```diff
@@ -16,5 +19,60 @@ type NodeScan struct {
 
 // Scan scans the current node and returns the results as storage.NodeInventory object
 func (n *NodeScan) Scan(nodeName string) (*storage.NodeInventory, error) {
-	return nil, errors.New("Not implemented")
+	componentsHost, err := nodes.Analyze(nodeName, "/host/", false)
+	log.Info("Finished node inventory /host scan")
+	if err != nil {
+		log.Errorf("Error scanning node /host inventory: %v", err)
+	}
+	log.Infof("Components found under /host: %v", componentsHost)
+	if err != nil {
+		return nil, err
+	}
+
+	var protoComponents *scannerV1.Components
+	if componentsHost != nil {
+		protoComponents = protoComponentsFromScanComponents(componentsHost)
+	}
```

Comment:
I suggest to handle `if` condition inside of `protoComponentsFromScanComponents` function.

```suggestion
	protoComponents = protoComponentsFromScanComponents(componentsHost)
```